src/renderer/widgets/SendAssetModal/ui/components/ActionSteps/InitOperation.tsx

```diff
@@ -204,7 +205,16 @@ export const InitOperation = ({
     const balance = balances.find((b) => b.accountId === account.accountId);
     const nativeBalance = nativeBalances.find((b) => b.accountId === account.accountId);
 
-    return getAccountOption(account, { addressPrefix, asset, amount, balance, nativeBalance, fee, deposit });
+    return getAccountOption(account, {
+      addressPrefix,
+      asset,
+      amount,
+      balance,
+      nativeBalance,
+      fee,
+      deposit,
+      isPolkadotVault: walletUtils.isPolkadotVault(activeWallet),
```

Comment:
And here you pass `hideName`, where `const hideName = walletUtils.isPolkadotVault(activeWallet) && accountUtils.isShardAccount(account)`